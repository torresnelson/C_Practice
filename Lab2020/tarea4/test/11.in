# Prueba de unión, diferencia e intersección entre conjuntos vacíos y singleton 
# vacío - vacío
unionDeConjuntos 0
iteradorDeConjunto
diferenciaDeConjuntos 0
iteradorDeConjunto
interseccionDeConjuntos 0
iteradorDeConjunto
# vacío - singleton
diferenciaDeConjuntos 1 3
iteradorDeConjunto
interseccionDeConjuntos 1 4
iteradorDeConjunto
unionDeConjuntos 1 5
iteradorDeConjunto
# singleton - vacío 
unionDeConjuntos 0
iteradorDeConjunto
diferenciaDeConjuntos 0
iteradorDeConjunto
interseccionDeConjuntos 0
iteradorDeConjunto
# para volver a tener un singleton
unionDeConjuntos 1 6
# singleton - singleton
unionDeConjuntos 1 6
iteradorDeConjunto
diferenciaDeConjuntos 1 3
iteradorDeConjunto
interseccionDeConjuntos 1 6
iteradorDeConjunto
diferenciaDeConjuntos 1 6
iteradorDeConjunto
unionDeConjuntos 1 6
interseccionDeConjuntos 1 3
iteradorDeConjunto
unionDeConjuntos 1 6
unionDeConjuntos 1 4
iteradorDeConjunto
# para volver a tener un singleton
interseccionDeConjuntos 1 6
iteradorDeConjunto
diferenciaDeConjuntos 1 8
iteradorDeConjunto
interseccionDeConjuntos 1 8
iteradorDeConjunto
# para volver a tener un singleton
unionDeConjuntos 1 6
unionDeConjuntos 1 8
iteradorDeConjunto
# posibles colisiones
asociarEnMap 0 -1.0
asociarEnMap 10 10.0
valorEnMap 0
valorEnMap 10
desasociarEnMap 10
asociarEnMap 11 11.0
valorEnMap 11
asociarEnMap 12 12.0
valorEnMap 12
asociarEnMap 13 13.0
valorEnMap 13
asociarEnMap 14 14.0
valorEnMap 14
asociarEnMap 15 15.0
valorEnMap 15
asociarEnMap 16 16.0
valorEnMap 16
asociarEnMap 17 17.0
valorEnMap 17
asociarEnMap 18 17.0
valorEnMap 18
asociarEnMap 19 19.0
valorEnMap 19
valorEnMap 0
desasociarEnMap 0
asociarEnMap 10 10.0
valorEnMap 10
valorEnMap 11
valorEnMap 12
valorEnMap 13
valorEnMap 14
valorEnMap 15
valorEnMap 16
valorEnMap 17
valorEnMap 18
valorEnMap 19
# actualizaciones que aumentan
insertarEnCP 3 3.0
prioritario
insertarEnCP 4 4.0
prioritario
insertarEnCP 2 2.0
prioritario
insertarEnCP 1 1.0
prioritario
actualizarEnCP 2 0
prioritario
actualizarEnCP 3 -1
prioritario
actualizarEnCP 1 5.0
actualizarEnCP 2 7.0
actualizarEnCP 3 6
prioritario
Fin
