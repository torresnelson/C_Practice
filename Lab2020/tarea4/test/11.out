1>#  Prueba de unión, diferencia e intersección entre conjuntos vacíos y singleton .
2>#  vacío - vacío.
3>Union.
4>No hay elementos en el conjunto.
5>Diferencia.
6>No hay elementos en el conjunto.
7>Intersección.
8>No hay elementos en el conjunto.
9>#  vacío - singleton.
10>Diferencia.
11>No hay elementos en el conjunto.
12>Intersección.
13>No hay elementos en el conjunto.
14>Union.
15>5 
16>#  singleton - vacío .
17>Union.
18>5 
19>Diferencia.
20>5 
21>Intersección.
22>No hay elementos en el conjunto.
23>#  para volver a tener un singleton.
24>Union.
25>#  singleton - singleton.
26>Union.
27>6 
28>Diferencia.
29>6 
30>Intersección.
31>6 
32>Diferencia.
33>No hay elementos en el conjunto.
34>Union.
35>Intersección.
36>No hay elementos en el conjunto.
37>Union.
38>Union.
39>4 6 
40>#  para volver a tener un singleton.
41>Intersección.
42>6 
43>Diferencia.
44>6 
45>Intersección.
46>No hay elementos en el conjunto.
47>#  para volver a tener un singleton.
48>Union.
49>Union.
50>6 8 
51>#  posibles colisiones.
52>Establecida la asociación.
53>Establecida la asociación.
54>-1.00
55>10.00
56>Eliminada la asociación.
57>Establecida la asociación.
58>11.00
59>Establecida la asociación.
60>12.00
61>Establecida la asociación.
62>13.00
63>Establecida la asociación.
64>14.00
65>Establecida la asociación.
66>15.00
67>Establecida la asociación.
68>16.00
69>Establecida la asociación.
70>17.00
71>Establecida la asociación.
72>17.00
73>Establecida la asociación.
74>19.00
75>-1.00
76>Eliminada la asociación.
77>Establecida la asociación.
78>10.00
79>11.00
80>12.00
81>13.00
82>14.00
83>15.00
84>16.00
85>17.00
86>17.00
87>19.00
88>#  actualizaciones que aumentan.
89>Se insertó el elemento.
90>3
91>Se insertó el elemento.
92>3
93>Se insertó el elemento.
94>2
95>Se insertó el elemento.
96>1
97>Se actualizó el valor.
98>2
99>Se actualizó el valor.
100>3
101>Se actualizó el valor.
102>Se actualizó el valor.
103>Se actualizó el valor.
104>4
105>Fin.
